pkg/controller/volume/persistentvolume/testing/testing.go

```diff
@@ -144,7 +145,7 @@ func (r *VolumeReactor) React(action core.Action) (handled bool, ret runtime.Obj
 		}
 		r.changedObjects = append(r.changedObjects, claim)
 		r.changedSinceLastSync++
-		klog.V(4).Infof("created claim %s", claim.Name)
+		logger.V(4).Info("Created claim", "claimName", claim.Name)
```

Comment:
Instead of "claimName", use "PVC" as above and log with `klog.KObj`.

Reply:
done